### DISCUSSION TITLE
Error decoding datasource marcadores.iped

### DISCUSSION THREAD

**Discussion by Klarissa-ksj** [sepinf-inc/IPED#1019] [Q&A] [answered]
Hello @lfcnassif, @tc-wleite and everybody,

I have some errors trying to reindex selected files. 

In my attached log I see that it searches for "prevTempDir.txt" in the wrong unity F:, that is where it is located IPED program, not the directory H:\ where it is located the case.

Adding 'H:\IPL xxxxx_SJK_SP_MAT.360\IPED_item01\indexador\marcadores.iped'

[ERROR]	[gpinf.indexer.IndexFiles]			Processing Error: 
java.lang.Exception: Error decoding datasource H:\IPL xxxxx_SJK_SP_MAT.360\IPED_item01\indexador\marcadores.iped

Caused by: java.lang.RuntimeException: java.nio.file.NoSuchFileException: F:\IPED_item01\indexador\data\**prevTempDir.txt**

Help me with configurations. It used relative path in this case of F:, but I should be 
H:\IPL xxxxx_SJK_SP_MAT.360\IPED_item01\indexador\data

Thanks
[IPED-2022-03-16-11-30-50.log](https://github.com/sepinf-inc/IPED/files/8265255/IPED-2022-03-16-11-30-50.log)

**Comment by lfcnassif** [comment, marked as answer]
Hi @Klarissa-ksj,

Have you moved your original case location from F:\IPED_item01 to H: after saving your bookamarks? If yes, you must update your marcadores.iped file with the new case location. Internally, that file keeps absolute references to the case folder, even with --portable option. Simply open the case and save the marcadores.iped file again.

**Reply by lfcnassif** [reply]
> save the marcadores.iped file again.

PS: just check and uncheck a random item, and it will be automatically saved

**Reply by lfcnassif** [reply]
> that file keeps absolute references to the case folder, even with --portable option

That is done to allow you saving that bookmarks file anywhere, and it will find the original case when generating the report.

**Reply by Klarissa-ksj** [reply]
No. I have not moved the case recently, but I analysed It in an IPED visualization of multiple cases (3 itens).
Then I saw that each marcadores.iped was updated.
When I arrive at office, I Will do your instructions and return here the results.

**Reply by Klarissa-ksj** [reply]
Solved.
I updated each marcadores.iped in each individual case instead of unified IPED and the I had sucess in reindexing selected files. 

Thank you.

**Reply by lfcnassif** [reply]
You're welcome
